packages/devtools_app/.gitignore

```diff
@@ -8,7 +8,6 @@ doc/api/
 .flutter-plugins-dependencies
 android/
 ios/
-lib/generated_plugin_registrant.dart
```

Comment:
add this line back and remove the changes to lib/generated_plugin_restraint below